{
  "$schema": "https://raw.githubusercontent.com/rwx-research/test-results-schema/main/v1.json",
  "framework": {
    "language": "Ruby",
    "kind": "Cucumber"
  },
  "summary": {
    "status": {
      "kind": "failed"
    },
    "tests": 3,
    "otherErrors": 1,
    "retries": 0,
    "canceled": 0,
    "failed": 0,
    "pended": 0,
    "quarantined": 0,
    "skipped": 3,
    "successful": 0,
    "timedOut": 0,
    "todo": 0
  },
  "tests": [
    {
      "name": "Rule Sample \u003e With a failing before hook \u003e this will pass",
      "lineage": [
        "Rule Sample",
        "With a failing before hook",
        "this will pass"
      ],
      "location": {
        "file": "features/rule.feature",
        "line": 37
      },
      "attempt": {
        "durationInNanoseconds": null,
        "meta": {
          "elementStart": "features/rule.feature:34",
          "stepLocation": "features/step_definitions/steps.rb:1",
          "tags": [
            {
              "name": "@feature_tag",
              "line": 1
            },
            {
              "name": "@rule_tag",
              "line": 4
            },
            {
              "name": "@failing_before_hook",
              "line": 33
            }
          ]
        },
        "status": {
          "kind": "skipped"
        }
      }
    },
    {
      "name": "Rule Sample \u003e With a failing before hook \u003e I do an action",
      "lineage": [
        "Rule Sample",
        "With a failing before hook",
        "I do an action"
      ],
      "location": {
        "file": "features/rule.feature",
        "line": 37
      },
      "attempt": {
        "durationInNanoseconds": null,
        "meta": {
          "elementStart": "features/rule.feature:34",
          "stepLocation": "features/step_definitions/steps.rb:9",
          "tags": [
            {
              "name": "@feature_tag",
              "line": 1
            },
            {
              "name": "@rule_tag",
              "line": 4
            },
            {
              "name": "@failing_before_hook",
              "line": 33
            }
          ]
        },
        "status": {
          "kind": "skipped"
        }
      }
    },
    {
      "name": "Rule Sample \u003e With a failing before hook \u003e some results should be there",
      "lineage": [
        "Rule Sample",
        "With a failing before hook",
        "some results should be there"
      ],
      "location": {
        "file": "features/rule.feature",
        "line": 37
      },
      "attempt": {
        "durationInNanoseconds": null,
        "meta": {
          "elementStart": "features/rule.feature:34",
          "stepLocation": "features/step_definitions/steps.rb:12",
          "tags": [
            {
              "name": "@feature_tag",
              "line": 1
            },
            {
              "name": "@rule_tag",
              "line": 4
            },
            {
              "name": "@failing_before_hook",
              "line": 33
            }
          ]
        },
        "status": {
          "kind": "skipped"
        }
      }
    }
  ],
  "otherErrors": [
    {
      "location": {
        "file": "features/step_definitions/steps.rb",
        "line": 24
      },
      "message": "failed in before hook (RuntimeError)\n./features/step_definitions/steps.rb:25:in `Before'",
      "meta": {
        "type": "before hook"
      }
    }
  ]
}
